src/redux/slices/areaSpecialization/slice.ts

```diff
@@ -0,0 +1,12 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+import { getInitialState } from './initialState'
+import reducers from './reducers'
+
+const slice = createSlice({
+  name: 'users',
+  initialState: getInitialState(),
+  reducers,
+})
+
```

Comment:
Why slice name is users but it's in areaSpecialization file, also I think we should rename this file

Reply:
done